puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-rpm.groovy

```diff
@@ -17,7 +17,7 @@ pipeline {
             agent { label 'sshkey' }
 
             steps {
-                mash('foreman-mash-split.py')
+                mash('foreman-mash-split.py', 'nightly')
```

Comment:
Why is this hard coded to nightly?

Reply:
because that's the nightly pipeline file and the script defaults to "nightly" if no params are given, I just wanted it to be explicit.

Reply:
Below we do call `repoclosure` with `env.foreman_version` which I think is `nightly` as well. Interestingly enough in a lot of other places we do hardcode nightly.